SPECS/python-botocore/python-botocore.spec

```diff
@@ -39,17 +37,19 @@ A low-level interface to a growing number of Amazon Web Services. The botocore p
 %py3_install
 
 %check
-pip3 install nose
-pip3 install mock
-pip3 install jmespath
-nosetests tests/unit
+pip3 install tox
+tox -e py39
```

Comment:
Why did we switch to tox here?

Reply:
tox is a modern testing framework and since this project supports it, it's only a matter of 'tox -e py39'.
It automatically sets up every dependency and does the right thing.
In my opinion, we should switch to tox for testing wherever it is supported.
The test that this instance of tox runs is exactly the same one that the official projects CI uses.